documentation/src/components/SearchModal/styles.module.css

```
@@ -0,0 +1,374 @@
+    stroke-linecap: round;
+    stroke-width: 1.8;
+}
+
+.navbarTrigger .triggerLabel {
+    gap: 0.5rem;
+}
+
+.navbarTrigger .triggerLabel svg {
+    width: 1rem;
+    height: 1rem;
+    stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)